save-demo/src/main/kotlin/com/saveourtool/save/demo/service/DiktatDemoService.kt

```diff
@@ -5,119 +5,62 @@ import com.saveourtool.save.demo.diktat.DiktatDemoMode
 import com.saveourtool.save.demo.diktat.DiktatDemoResult
 import com.saveourtool.save.demo.diktat.DiktatDemoTool
 
-import com.pinterest.ktlint.core.KtLint
-import com.pinterest.ktlint.core.LintError
-import com.pinterest.ktlint.core.RuleSet
-import com.pinterest.ktlint.ruleset.standard.StandardRuleSetProvider
-import org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider
+import com.saveourtool.save.demo.runners.DiktatCliRunner
 import org.springframework.stereotype.Service
 
 import java.io.File
 import java.util.*
 
-import kotlin.collections.ArrayList
-
 /**
  * Demo service implementation for ktlint-demo/diktat-demo
  */
 @Service
 class DiktatDemoService : AbstractDemoService<DiktatDemoAdditionalParams, DiktatDemoResult> {
-    private fun generateFileName() = UUID.randomUUID().toString()
-    private fun generateDemoFile(generatedName: String): File = File("demo-input-$generatedName")
-    private fun generateDemoConfig(generatedName: String): File = File("demo-config-$generatedName")
+    private fun generateName() = UUID.randomUUID().toString()
```

Comment:
suggestion to append `tmpDir` and set it via property

Reply:
This is what will be done in the next PR. Also it is planned to use file based storage in order to store all the executables